test/packages/apache/_dev/deploy/variants.yml

```diff
@@ -1,6 +1,6 @@
 variants:
   v2420:
     SERVICE_VERSION: 2.4.20
-  v2443:
-    SERVICE_VERSION: 2.4.43
+  v2423:
+    SERVICE_VERSION: 2.4.23
```

Comment:
Have we considered to add something to skip variants? This may be needed in the future.

Reply:
We have the "skip" option for the entire test config not for particular variant. Maybe a follow up issue?

Reply:
:+1: